### PR TITLE
added create contact page

### PR DESCRIPTION
**Pull request by Smita-92**
please review

### CODE REVIEW

File: deno_git_practise/src/test/java/trialpackage/createcontact.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package trialpackage;
+
+public class createcontact {
+
+	public static void main(String[] args) {
+		System.out.println("create new contact");
+
```
Comment (Smita-92) [inline deno_git_practise/src/test/java/trialpackage/createcontact.java:7]: change the xpath.dont hardcode the data

**Review by Smita-92 — COMMENTED** [review]
added few comments

File: deno_git_practise/src/test/java/trialpackage/createcontact.java (hunk 1 above)
Comment (Smita-92) [reply deno_git_practise/src/test/java/trialpackage/createcontact.java:7]: fixed

**Comment by Smita-92** [comment]
accepted

**Comment by DebasmitaBehera** [comment]
merged

**Review by DebasmitaBehera — COMMENTED** [review]
ok good work

**Pull request merged**